v2x/vlog_to_pbtype.py

```diff
@@ -54,6 +54,27 @@
     - `(* carry = ADDER *)` : specify carry chain pack_pattern associated
                               with this port
 
+The following attributes are used to annotate cells with fasm metadata:
+    - `(* FASM_PREFIX="prefix" *)` : Sets the fasm prefix of a cell instance.
+        Cannot be set on a module definition!
+
+    - `(* FASM_PREFIX="prefix1;prefix2;..." *) : Same as FASM_PREFIX but used
```

Comment:
I'm less happy with this extension.

Reply:
Honestly I have no better idea how to deal with this use case.

Reply:
@mkurc-ant - ping?

Reply:
@mithro How would you suggest to set fasm prefix on cell instances created using a `generate` statement? I couldn't think of anything better than `(* FASM_PREFIX="prefix1;prefix2;..." *)`

Reply:
@mithro ping?